.env

```diff
@@ -85,3 +85,4 @@ NEXT_PUBLIC_CAMPAIGN_IS_PREVIEW=false
 REDIRECT_URL_FOR_IMPORT=$SITE_URL
 BREVO_ALLOWED_REDIRECT_URL=http://${DEV_DOMAIN:-localhost}${WORKAROUND_DOTENV_ISSUE}:${SITE_PORT}
 CAMPAIGNS_FRONTEND_URL=http://${DEV_DOMAIN:-localhost}${WORKAROUND_DOTENV_ISSUE}:${SITE_PORT} # doesn't work, TODO: add actual mailing frontend
+BREVO_FOLDER_ID=
```

Comment:
I would either set it to 1 here or not set it at all

Reply:
ae8df102dbdbce0e39aa40ece8ff10f796cef326